.github/workflows/_testing.yml

```diff
@@ -25,19 +25,25 @@ jobs:
   specific-version:
     uses: ./.github/workflows/ci.yml
     with:
-      flink_version: 1.16.1
+      flink_version: 1.17.1
       connector_branch: ci_utils
-  snapshot-version:
+  snapshot-without-sanity-checks:
     uses: ./.github/workflows/ci.yml
     with:
-      flink_version: 1.16-SNAPSHOT
-      connector_branch: ci_utils
-  disable-convergence:
+      flink_version: 1.19-SNAPSHOT
+      run_sanity_checks: false
+  non-main-version_without-sanity-checks:
     uses: ./.github/workflows/ci.yml
     with:
-      flink_version: 1.16.1
-      connector_branch: ci_utils
```

Comment:
Why is this being removed?

Reply:
just for testing as the workflow is called "testing" I though I could use it to test my setup :smile: